website/docs/r/bigtable_table.html.markdown

```diff
@@ -41,9 +41,17 @@ The following arguments are supported:
 
 * `split_keys` - (Optional) A list of predefined keys to split the table on.
 
+* `column_family` - (Optional) A block of column family configuration options.
```

Comment:
What about this? Mostly cribbed from [the REST API](https://cloud.google.com/bigtable/docs/reference/admin/rest/v2/projects.instances.tables/modifyColumnFamilies#ColumnFamily).

```
A group of columns within a table which share a common configuration. This can be specified multiple times. Structure is documented below.
```

Reply:
Yes better, updated too!